### ISSUE TITLE
Where clause replicated incorrectly

### ISSUE THREAD

**Issue by bhoot929** [darold/ora2pg#521]
**Below is the original query:**
```
with INVOICE AS 
(
    SELECT 
INV.YYYYMM,
INV.MNTH,
INV.LEGAL_ENTITY_ID,
INV.ORG,
INV.INVOICE_VOLUME ACTUAL_VOL,
AVG(THREE.INVOICE_VOLUME) AVG_VOL
    FROM 
INV_CNT INV,INV_CNT THREE
    WHERE 
INV.ORG=THREE.ORG
    AND THREE.MNTH >= ADD_MONTHS(TRUNC(INV.MNTH,'MM'),-2) 
AND THREE.MNTH <= INV.MNTH
    group by INV.YYYYMM,INV.MNTH,INV.LEGAL_ENTITY_ID,INV.ORG,INV.INVOICE_VOLUME
HAVING COUNT(*) =3
)
select 
'AP' as GRP,
'MONTH' PERIOD,
invoice.YYYYMM as YYYYMM,
invoice.mnth as mnth,
invoice.legal_entity_id as legal_entity_id,
invoice.org as ORG,
invoice.avg_vol as invoice_volume,
fincom_case.case_count as CASE_COUNT,
(1000/invoice.AVG_VOL)*fincom_case.case_count AS CONTACTS4_1KINVOICES
from
INVOICE,
(
    SELECT 
TO_CHAR(ctr.activity_date,'YYYY-MM') AS YYYYMM,
TRUNC(ctr.activity_date,'MM') MONTH,
    Q.org,
SUM(CTR.createdinbound+CTR.createdoutbound+CTR.transferredin) CASE_COUNT
    FROM 
ppv_op_ddl.yuma_case_snapshot_ctr ctr,
    ppv_op_ddl.yuma_case_queue_org_map q
    where 
q.logical_queue_name=ctr.queue_name
    and q.dept  ='AP' 
and TRUNC(ctr.activity_date,'MM') >=ADD_MONTHS(TRUNC(SYSDATE,'MM'),-11)
    GROUP BY 
TO_CHAR(activity_date,'YYYY-MM'),
TRUNC(ctr.activity_date,'MM'),
    q.org
) fincom_case
where 
invoice.yyyymm      =fincom_case.yyyymm
AND INVOICE.org     =fincom_case.org
```

**Converted:**
```
with INVOICE AS (
    SELECT
INV.YYYYMM,
INV.MNTH,
INV.LEGAL_ENTITY_ID,
INV.ORG,
INV.INVOICE_VOLUME ACTUAL_VOL,
AVG(THREE.INVOICE_VOLUME) AVG_VOL
    FROM 
INV_CNT INV,INV_CNT THREE
    WHERE 
INV.ORG=THREE.ORG
    AND THREE.MNTH >= TRUNC(INV.MNTH,'MM') + -2*'1 month'::interval 
AND THREE.MNTH <= INV.MNTH
    group by INV.YYYYMM,INV.MNTH,INV.LEGAL_ENTITY_ID,INV.ORG,INV.INVOICE_VOLUME
GROUP BY 
TO_CHAR(activity_date,'YYYY-MM'),
date_trunc('month', ctr.activity_date),
    q.org
) fincom_case
**where 
invoice.yyyymm      =fincom_case.yyyymm
AND INVOICE.org     =fincom_case.org HAVING COUNT(*) =3
)**
select 
'AP' as GRP,
'MONTH' PERIOD,
invoice.YYYYMM as YYYYMM,
invoice.mnth as mnth,
invoice.legal_entity_id as legal_entity_id,
invoice.org as ORG,
invoice.avg_vol as invoice_volume,
fincom_case.case_count as CASE_COUNT,
(1000/invoice.AVG_VOL)*fincom_case.case_count AS CONTACTS4_1KINVOICES
from
INVOICE,
(
    SELECT 
TO_CHAR(ctr.activity_date,'YYYY-MM') AS YYYYMM,
date_trunc('month', ctr.activity_date) MONTH,
    Q.org,
SUM(CTR.createdinbound+CTR.createdoutbound+CTR.transferredin) CASE_COUNT
    FROM 
ppv_op_ddl.yuma_case_snapshot_ctr ctr,
    ppv_op_ddl.yuma_case_queue_org_map q
    where 
q.logical_queue_name=ctr.queue_name
    and q.dept  ='AP' 
and date_trunc('month', ctr.activity_date) >=date_trunc('month', LOCALTIMESTAMP) + -11*'1 month'::interval
;
```

In converted query, I have highlighted the code which was replicated incorrectly.
Can you please check.

**Comment by darold** [comment]
Might be fixed in last commit 6f54e81.

**Comment by bhoot929** [comment]
Its resolved.Thanks
